lib/cli/test/unit/Cardano/CLISpec.hs

```diff
@@ -147,6 +204,19 @@ spec = do
             , "!!! Only for the Incentivized Testnet !!!"
             ]
 
+
+        ["key", "--help"] `shouldShowUsage`
+            [ "Usage:  key COMMAND"
+            , "  Derive keys from mnemonics."
+            , ""
+            , "Available options:"
+            , "  -h,--help                Show this help text"
+            , ""
+            , "Available commands:"
+            , "  root                     Extract root xprv as hex (64 bytes"
```

Comment:
Extract root extended private key from a mnemonic sentence* 

would be better. 

a) Avoid "cryptic" abbreviation in public-facing interface. `xprv` sounds familiar to us, but users external to the project have no idea what this is about. 
b) Don't mention "hex" here, that's simply the encoding format and is relatively irrelevant (the command-line could even support multiple encoding format). 
c) It does say from where / what the key is extracted.